DLplatform/coordinator.py

```diff
@@ -236,12 +236,19 @@ def onMessageReceived(self, routing_key, exchange, body):
             # we send around the initial parameters only when all the expected nodes are there
             # in case when parameter is not set, it is equal to 0 - so every new node will satisfy the condition
             if len(self._waitingNodes) >= self._minStartNodes:
+                for id in self._waitingNodes:
```

Comment:
can you please update it in a way that is currently in the master branch? there were some changes that this update will override. I guess overall you were not changing coordinator.py? so it should be equivalent to the original

Reply:
I have merged in the recent commits from the Fraunhofer master branch. So there wont be any conflicts